qml/nextcloud/NewAccount.qml

```diff
@@ -190,12 +191,16 @@ Item {
                         if (statusCode == "999") {
                             showError(__hostError)
                         } else {
-                            showError(i18n.dtr("account-plugins", "Invalid username or password"))
+                            showError(i18n.dtr("account-plugins", __usernameError))
                         }
                         callback(false)
                     }
                 } else {
-                    showError(__hostError)
+                    if (request.status == 401) {
+                        showError(i18n.dtr("account-plugins", __usernameError))
```

Comment:
`__usernameError` is already translated, so here you should not use `i18n.dtr`.

Reply:
Oops. I had the string there first and then replaced it by the variable... Is fixed by #19